website/docs/how-to/using-a-capacitor-plugin.md

```diff
@@ -2,3 +2,276 @@
 title: How To Use a Capacitor Plugin
 sidebar_label: Use a Capacitor Plugin
 ---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Ionic Portals uses Capacitor under the hood, meaning that you can use existing Capacitor Plugins in your Portals. These plugins allow Portals to use native functionality without much setup on either the native or web developers part.
+
+<Tabs 
+    defaultValue="ios" 
+    values={[
+        { label: 'iOS', value: 'ios', },
+        { label: 'Android', value: 'android', },
+    ]}
+>
+<TabItem value="ios">
+
+## Native Usage
+In order to use a Capacitor Plugin, you need to install the plugin as a dependency in your `Podfile`.
+
+```ruby
```

Comment:
change this to ```ruby Podfile {2)``` and take out the podfile comment. The {2} will highlight the line for them to add so it calls it out better.